next.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/page/1',
+        permanent: true
+      }
+    ];
+  }
```

Comment:
It ok, but I really not enjoy this.

Reply:
this is to make the default redirect from home page to `/page/1`

do you have any suggestion to make it looks better?